### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by ebell495**

### CODE REVIEW

File: Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+project: word-checker
```
Comment (xansec) [inline Mayhemfile:1]: Can we rename this target to indicate what's being fuzzed?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: Mayhemfile (hunk 1 above)
Comment (ebell495) [reply Mayhemfile:1]: I've renamed it

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**